.pre-commit-config.yaml

```diff
@@ -42,7 +42,7 @@ repos:
   rev: v2.2.2
   hooks:
   - id: codespell
-    args: ["--skip=*.pyc,tests/testdata/*,*.ipynb,*.csv"]
+    args: ["--skip=*.pyc,tests/testdata/*,*.ipynb,*.csv","--ignore-words-list=reacher"]
```

Comment:
We used to have `iff` skipped in `.codespell.skip` -- is that not needed?

We should probably also delete `.codespell.skip` if no longer used.

Reply:
Done.